eng/scripts/DownloadDevPackages.ps1

```diff
@@ -0,0 +1,35 @@
+param (
+    [Parameter(Mandatory=$True)]
+    [string] $WorkingDirectory,
+    [Parameter(Mandatory=$True)]
+    [string] $NupkgDestination,
+    [string] $NugetSource="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json",
+    [string] $FallBackSource = "https://api.nuget.org/v3/index.json"
+)
+
+. (Join-Path $PSScriptRoot ".." common scripts common.ps1)
+
+$allPackages = Get-AllPkgProperties
+$trackTwoPackages = $allPackages.Where{ $_.SdkType -eq "client" }
+
+Push-Location $WorkingDirectory
+$nugetPackagesPath = Join-Path $WorkingDirectory nugetPackages
+New-Item -Path $WorkingDirectory -Type "directory" -Name "nugetPackages"
+
+foreach ($package in $trackTwoPackages)
+{
+  nuget install $package.Name `
+  -OutputDirectory $nugetPackagesPath `
+  -Prerelease `
+  -Source $NugetSource `
+  -FallBackSource $FallBackSource `
+  -DependencyVersion Ignore `
+  -DirectDownload `
+  -NoCache `
+  -ExcludeVersion
+}
+
+$nupkgFilesPath = Join-Path $WorkingDirectory $NupkgDestination
```

Comment:
Have you confirmed that this only puts the packages we are interested in in the path?

Reply:
Yes, Only packages marked as IsNewSdk are downloaded without their dependencies  and this just copies the nupkg file out to where it can be used for other processes.